Sources/MapboxSearch/PublicAPI/Search Results/Parking/ParkingRate.swift

```diff
@@ -0,0 +1,24 @@
+@_spi(Experimental)
+public struct ParkingRate: Codable, Hashable {
+    public let maxStayISO8601: String?
```

Comment:
Could it be a timestamp? it's not convinient for the customers to operate with strings, especially if we don't specify the timezone, etc.